src/components/Buttons/LinkButton.tsx

```diff
@@ -1,31 +1,21 @@
-"use client";
-
 import React from "react";
-import Link from "next/link";
-import styled from "styled-components";
 import Button from "@mui/material/Button";
-import { usePathname } from "next/navigation";
-
-const UnstyledLink = styled(Link)`
-    text-decoration: none;
-    display: contents;
-`;
+import { ButtonWrap, UnstyledLink } from "@/components/Buttons/GeneralButtonParts";
 
-type LinkButtonProps = {
+interface Props {
+    children: string;
     link: string;
-    page: string;
-    onClick?: () => void;
-};
+}
 
-const LinkButton: React.FC<LinkButtonProps> = (props) => {
-    const pathname = usePathname();
-    const active = pathname?.startsWith(props.link) ?? false;
+const LinkButton: React.FC<Props> = ({ children, link }) => {
     return (
-        <UnstyledLink key={props.page} href={props.link} prefetch={false}>
-            <Button color="primary" variant={active ? "contained" : "outlined"}>
-                {props.page}
-            </Button>
-        </UnstyledLink>
+        <ButtonWrap>
```

Comment:
Could we just style the button instead of adding another div into the dom?

Reply:
Seems like some buttons don't use the wrap though

Reply:
this button wrap is to add margin so you can't directly style the button for that (especially since it's wrapped inside a link)

Reply:
I am thinking maybe it is also possible to pass in margin as a boolean prop rather than a separate div (?)

Reply:
can you not add the margin on the UnstyledLink (and only have ButtonWrap for PopUpButton, meaning it can live in the same file and not have to be exported)?